internal/storage/s3/s3.go

```diff
@@ -0,0 +1,139 @@
+package s3
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"path/filepath"
+	"time"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	strg "github.com/offen/docker-volume-backup/internal/storage"
+	t "github.com/offen/docker-volume-backup/internal/types"
```

Comment:
Using single letter identifiers for packages is kind of unidiomatic in Go. If there is no hard reason (e.g. collisions) to rename an import, it's probably best to use the original package name.